### DISCUSSION TITLE
命令行-u指定时，貌似配置文件naming不生效

### DISCUSSION THREAD

**Discussion by zpq5935** [Johnserf-Seed/TikTokDownload#592] [Q&A]
命令行-u指定时，貌似配置文件naming不生效

**Comment by Johnserf-Seed** [comment]
当前版本使用命令行是不读取配置的，要使用--naming指定

**Comment by zpq5935** [comment]
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/44501567/8aae12d4-28b7-43a7-b97f-2a587edaefa5)

哦哦 我找到有个地方有这个使用说明 不过windows下好像没有这个说明帮助是吗

**Reply by Johnserf-Seed** [reply]
命令行是只能在终端里查看帮助信息，和操作系统无关
